jobspec.yaml

```diff
@@ -1,30 +1,30 @@
 spec:
   declarative: true
-  git_url: https://gitlab.cee.redhat.com/coreos/rhcos-upshift.git
-  git_ref: master
+  git_url: https://github.com/openshift/ocp-build-data
+  git_ref: rhcos-4.1
 cosa:
   override: false
   image: coreos-assembler:latest
 job:
-  build_name: ben-ootpa
-  brew_profile: brew-stage
+  build_name: rhcos-4.1
+  brew_profile: brew
   dry_run: false
-  init: true
+  init: false
   force: false
-  is_production: false
+  is_production: true
   skip_kola: true
   os_name: ootpa
   pipeline_yaml: src/config/buildparams.yaml
 recipe:
   os_name: ootpa
-  git_ref: master
+  git_ref: 4eb5e6c74f7d3ade81966d6707caf88c6d3cb538
```

Comment:
This is the current `master` HEAD right? We should port over to the ART branch and use that ref?

Reply:
by specifying a git ref, moving to a newer version of the pipeline is reflected here in this config, thus auditable in git history.

Reply:
Right what I mean is this is a hash not in the ART branch but only the master branch. That's fine, I'm just thinking if we should just explicitly remove the ART branch in gitlab then and just pin to commits like this.